packages/block-library/src/file/save.js

```diff
@@ -25,7 +25,9 @@ export default function save( { attributes } ) {
 		previewHeight,
 	} = attributes;
 
-	const pdfEmbedLabel = RichText.isEmpty( fileName ) ? 'PDF embed' : fileName;
+	const pdfEmbedLabel = RichText.isEmpty( fileName )
+		? 'PDF embed'
+		: fileName.toString();
```

Comment:
any idea why a filename is a rich text value and not a plaintext value?

Reply:
I have no idea :) It does seem a bit weird...

Reply:
But I'm also not sure if it's worth changing it. The Button block also allows formatting, so even though it's weird, it also doesn't hurt I guess?